### PR TITLE
Add stdin Signal

### PR DESCRIPTION
**Pull request by deadfoxygrandpa**
Add `Console.stdin : Signal String`

### CODE REVIEW

**Comment by deadfoxygrandpa** [comment]
This addresses issue #2

File: src/Native/Console.js [hunk 1]
```diff
@@ -10,10 +10,22 @@ Elm.Native.Console.make = function(localRuntime) {
...
 
+	var readline = require('readline');
+	var rl = readline.createInterface({
+		input: process.stdin,
+	});
+	
+	var stdin = NS.input("stdin", "");
+	
+	rl.on('line', function(line) {
+		localRuntime.notify(std.id, line);
```
Comment (ajhager) [inline src/Native/Console.js]: Where does std.id come from here?

Comment (deadfoxygrandpa) [inline src/Native/Console.js]: Oops, that's a typo that I somehow fixed locally but forgot to commit. It's supposed to say `stdin.id` and it's a property of the `stdin` signal as creating by `Elm.Native.Signal.input`

**Comment by ajhager** [comment]
LGTM!

**Comment by texastoland** [comment]
[LGTM](http://lgtm.in/i/0wAv7RPIn)
Stoked to see this in action :+1:

**Pull request merged**